app/src/main/java/com/macbitsgoa/ard/utils/HandleFirebaseMessages.java

```diff
@@ -0,0 +1,66 @@
+package com.macbitsgoa.ard.utils;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.util.Log;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Map;
+
+public class HandleFirebaseMessages extends FirebaseMessagingService {
+
+    public static Bitmap getBitmapFromURL(String src) {
+        try {
```

Comment:
A library solely designed for image loading will anyday be better at handling images and cache than our custom methods.
True you can get bitmap, but it would be better to let library manage caching, loading, memory management, anyways it is not like Fresco where getting the underlying bitmap is a pain..
use Glide directly. no need for above mentioned suggestion of using AHC